include/ck_cc.h

```diff
@@ -86,6 +86,12 @@
 #define CK_CC_UNLIKELY(x) x
 #endif
 
+#ifdef CK_CC_F_TYPEOF
+#define CK_CC_TYPEOF(X, DEFAULT) CK_CC_F_TYPEOF((X))
+#else
+#define CK_CC_TYPEOF(X, DEFAULT) (DEFAULT)
```

Comment:
All feature flags are under CK_F namespace. So, CK_F_PR_*, CK_F_CC_*, and so forth. Please make sure to reflect that (yes, it's a bit awkward but makes some local scripts easier to write without explicit enumeration).